EventFilter/SiStripRawToDigi/plugins/SiStripDigiToRaw.cc

```diff
@@ -134,7 +136,7 @@ namespace sistrip {
 
           //need to construct full object to copy full header
           if (rawfedData.size() == 0)
-            warnings_.add("Invalid raw data for FED, skipping", (boost::format("id %1%") % *ifed).str());
+            warnings_.add("Invalid raw data for FED, skipping", fmt::sprintf("id %1%", *ifed));
```

Comment:
actually, here it might be better to use
```suggestion
            warnings_.add("Invalid raw data for FED, skipping", fmt::format("id {0}", *ifed));
```